apps/core/tests/utils.py

```diff
@@ -0,0 +1,44 @@
+"""
+Usefull stuff for unittest
```

Comment:
s/Usefull/Useful
s/unittest/testing  (technically not unit tests but integration tests)